### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/CounterFit/commit/da714c1e5d765f2b5ea9a2b6f1ee2dfe9758a948#commitcomment-88313503

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/CounterFit/commit/1a6f36ab922c125ccc65fa662a45c3eabd165c4c

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: shims/CounterFitConnection/setup.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 '''
-Connection library library for use with the CounterFit Virtual IoT Device app
+Connection library for use with the CounterFit Virtual IoT Device app
```
Comment (jsoref) [inline shims/CounterFitConnection/setup.py:2]: I'm not actually confident about this...

File: shims/picamera/README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ To use these shims, you will need to install [CounterFit](https://github.com/Cou
     CounterFitConnection.init('127.0.0.1', 5000)
     ```
 
-* Write your Picamers code as usual.
+* Write your Picamera code as usual.
```
Comment (jsoref) [inline shims/picamera/README.md:32]: I'm hoping this is a brand...

File: shims/SeeedStudios/Seeed_Python_DHT/setup.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
     name='counterfit-shims-seeed-python-dht',
     py_modules=['counterfit_shims_seeed_python_dht'],
     version='0.1.0.dev1',
-    description='Shims for the Seeed Grove DBT for the CounterFit virtual IoT device app',
+    description='Shims for the Seeed Grove DHT for the CounterFit virtual IoT device app',
```
Comment (jsoref) [inline shims/SeeedStudios/Seeed_Python_DHT/setup.py:18]: This wasn't automatic, but it seemed likely, so, I'm asking...

**Review by jsoref — COMMENTED** [review]
Some corrections suggested by Google Sheets. All fault mine.

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**